ansible/roles/debops.postfix_virtual_ldap/defaults/main.yml

```diff
@@ -0,0 +1,685 @@
+# email-recipient to a username. Postfix first performs a full lookup
+# on user@domain, then user and then @domain.
+# The later one is also used for catchalls where the mailAlias
+# is set to e.g. @foobar.com
+postfix_virtual_ldap__postfix__lookup_virtual_senders:
+  search_base: '{{ postfix_virtual_ldap__ldap_people_dn|join(",") }}'
+  query_filter: '(&(mailAlias=%s){{ postfix_virtual_ldap__ldap_user_filter }})'
+  result_attribute: 'uid, mail'
+                                                                   # ]]]
+# .. envvar:: postfix_virtual_ldap__postfix__lookup_tables [[[
+#
+# The Virtual Mail lookup tables for postfix
+postfix_virtual_ldap__postfix__lookup_tables:
+  - name: 'ldap_virtual_aliases.cf'
+    state: 'present'
+    comment: |
+      The virtual_alias_maps setting is used to find the target mailbox
+      (configured in the attribute mail, aka the mailbox or maildrop).
+    group: 'postfix'
+    mode: '0640'
+    no_log: '{{ postfix_virtual_ldap__no_log|d(True) }}'
+    config: '{{ postfix_virtual_ldap__postfix__lookup_header |
+                 combine(postfix_virtual_ldap__postfix__lookup_virtual_aliases) }}'
+  - name: 'ldap_virtual_recipients.cf'
+    state: 'present'
+    comment: |
+      As we only want to accept mail, where we know the recipients
+      (and are responsible for them), the virtual_mailbox_maps configuration is used.
+    group: 'postfix'
+    mode: '0640'
+    no_log: '{{ postfix_virtual_ldap__no_log|d(True) }}'
+    config: '{{ postfix_virtual_ldap__postfix__lookup_header |
+                 combine(postfix_virtual_ldap__postfix__lookup_virtual_recipients) }}'
+  - name: 'ldap_virtual_domains.cf'
+    state: 'present'
+    comment: |
+      The virtual_mailbox_domains configurations performs a lookup,
+      if the postfix is responsible for the given domain.
+    group: 'postfix'
+    mode: '0640'
+    no_log: '{{ postfix_virtual_ldap__no_log|d(True) }}'
+    config: '{{ postfix_virtual_ldap__postfix__lookup_header |
+                 combine(postfix_virtual_ldap__postfix__lookup_virtual_domains) }}'
+  - name: 'ldap_virtual_senders.cf'
+    state: 'present'
+    comment: |
+      The smtpd_sender_login_maps configurations performs a lookup from an incoming
+      email-recipient to a username. Postfix first performs a full lookup
+      on user@domain, then user and then @domain.
+      The later one is also used for catchalls where the mailAlias
+      is set to e.g. @foobar.com
+    group: 'postfix'
+    mode: '0640'
+    no_log: '{{ postfix_virtual_ldap__no_log|d(True) }}'
+    config: '{{ postfix_virtual_ldap__postfix__lookup_header |
+                 combine(postfix_virtual_ldap__postfix__lookup_virtual_senders) }}'
+                                                                   # ]]]
+                                                                   # ]]]
+# LDAP [[[
+# ----------------------
+
+# LDAP authentication [[[
+# ^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I think that these sections should be on the same level as the `LDAP` section, you don't need to put them on yet another level. Just have two or more sections related to LDAP configuration.

Reply:
I like to have this hierarchy of sections to keep the topic

Reply:
Well, OK then. I'll have to see the rendered documentation to properly judge it. :-)